### PR TITLE
Fall back

### PR DESCRIPTION
**Pull request by kazu-yamamoto**
For #836, I would open this PR.

Please discuss whether or not we should use "SHOULD" here.

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -2612,6 +2625,11 @@ facilitates implementation as described below). Servers MUST check
 that it is the last extension and otherwise fail the handshake with an
 "illegal_parameter" alert.
 
+When the syntax of this extension is correct but the server cannot
+verify at least one binder because of some configuration or state
+changes, the server SHOULD fall back to a full 1-RTT handshake if a
+"key_share" extension is supplied.
```
Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: If the binder is present and the server believes that it has a key, then it MUST fail the handshake (in my view).  That's certainly what NSS does.  If the binder is present, but the server doesn't have a key, then it doesn't have a key and can't resume.

Comment (ekr) [inline draft-ietf-tls-tls13.md]: I agree with MT. The order of operations is:

- Determine whether you are going to PSK and select the PSK
- If you are resuming, check the binder.

If the binder checks fails, fail the handshake

Comment (kazu-yamamoto) [inline draft-ietf-tls-tls13.md]: Ah, I wrote this PR with confusion. The binder MUST be verified anyway.
What I should have talked about the encrypted PSK value itself.
I will push a fix.
Thank you for pointing out.

**Comment by kazu-yamamoto** [comment]
I pushed a commit to fix the condition. What about this?

**Comment by kazu-yamamoto** [comment]
I'm very sorry.
This branch accidentally contains my local commits.
So, I close this and resubmit a new PR.

**Comment by kazu-yamamoto** [comment]
@martinthomson @ekr Please read #894.

**Comment by davegarrett** [comment]
For future reference, you can simply force push a replacement set of commits to the same remote branch to change the contents of the PR, as needed. You don't have to refile completely.

**Comment by kazu-yamamoto** [comment]
@davegarrett Thanks!

**Pull request closed without merging**